gn3/db/genotypes.py

```diff
@@ -0,0 +1,184 @@
+"""Genotype utilities"""
+
+import os
+import gzip
+from typing import Union, TextIO
+
+from gn3.settings import GENOTYPE_FILES
+
+def build_genotype_file(
+        geno_name: str, base_dir: str = GENOTYPE_FILES,
+        extension: str = "geno"):
```

Comment:
*suggestion:* You could also  type-hint the return type:
```suggestion
        extension: str = "geno") -> str:
```